src/screens/DiscoveryScreenV2/fragment/UserFragmentCommunity.js

```diff
@@ -230,7 +230,7 @@ const UsersFragmentCommunity = ({
         data={data || []}
         renderItem={renderItem}
         keyExtractor={(item, index) => index.toString()}
-        onEndReached={() => fetchData()}
+        onEndReached={() => (data.length > 0 ? fetchData() : null)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It adds a check to ensure that `fetchData()` is only called when there are items in the `data` array. This can prevent unnecessary network requests when there's no data to fetch. However, it would be better to use a more explicit comparison against an empty array rather than checking if `data.length > 0`. This way, we can avoid potential type coercion issues and make the code more readable.

```diff
- onEndReached={() => (data.length > 0 ? fetchData() : null)}
+ onEndReached={() => (data && data.length ? fetchData() : null)}
```
This change ensures that `fetchData()` is only called when `data` is defined and has a length greater than zero.


<!-- This is an auto-generated comment by OSS CodeRabbit -->